server/db/models/Product.js

```diff
@@ -1,12 +1,48 @@
 const { STRING } = require('sequelize')
 const db = require('../db')
 
+const Product = conn.define('product', {
+    title: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: true,a
+        },
+    },
+    brand: {
+        type: STRING,
+        allowNull: false,
+    },
+    category: {
+        type: STRING,
+        validate: {
+            isIn: [['sweet', 'salty', 'healthy']],
+        },
+    },
+    description: {
+        type: Text,
```

Comment:
You need to import Text the same way you did with STRING, or just import DataTypes. Importing DataTypes and using it would look something like this:

DataTypes.STRING
DataTypes.TEXT

Reply:
Done!Thank you!